### PR TITLE
Ampeg selenium project

### PR DESCRIPTION
**Pull request by tchung7**

### CODE REVIEW

File: ampeg/src/test/java/AccountLoginPage.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.openqa.selenium.WebElement;
+
+public class AccountLoginPage extends PageObject {
+
+	public AccountLoginPage(WebDriver driver) {
+		super(driver);
+	}
+
+	public PostLoginTools login(String user, String pwd) {
+		WebElement username = driver.findElement(By.cssSelector("[id='l_email_user']"));
```
Comment (dentaylor) [inline ampeg/src/test/java/AccountLoginPage.java]: Be consistent with var usage. Line 18 uses it but this line does not.

File: ampeg/src/test/java/AccountLoginTests.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+
+public class AccountLoginTests extends AmpegTests {
+	@Test
+	public void ensureLoginIsSuccessful() {
+		var expectedUrl = "https://ampeg.com/account/";
+
+		var actualUrl = new HomePage(this.driver).clickSignInButton().login("miso9", "40nlyM32kn0!")
```
Comment (dentaylor) [inline ampeg/src/test/java/AccountLoginTests.java]: In tests, put each step on its own line.

Comment (dentaylor) [inline ampeg/src/test/java/AccountLoginTests.java]: Move the user name and password parameters to variables. This allows your code to be self documenting.

File: ampeg/src/test/java/ManualDownloadsPageTests.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+
+public class ManualDownloadsPageTests extends AmpegTests {
+	@Test
+	public void ensurev4BManualpdfLoads() {
+		var expectedurl = "https://ampeg.com/data/6/0a020a4177de6008ae7026058/application/pdf/Quick%20Start%20Guide%20-%20English%20.pdf";
```
Comment (dentaylor) [inline ampeg/src/test/java/ManualDownloadsPageTests.java]: 0a020a4177de6008ae7026058
This appears to be a dynamic value. Are you sure it won't change?

**Review by dentaylor — DISMISSED** [review]

File: ampeg/src/test/java/ManualDownloadsPageTests.java (hunk 3 above)
Comment (tchung7) [reply ampeg/src/test/java/ManualDownloadsPageTests.java]: hmmm, i see your point. Should I deprecate expectedurl and just put this.driver.getCurrentUrl() in the assertion?

Comment (dentaylor) [reply ampeg/src/test/java/ManualDownloadsPageTests.java]: I would assert on the file name since it probably will not change.

**Review by dentaylor — APPROVED** [review]

**Pull request merged**